### ISSUE TITLE
Use locksroot from TokenNetwork.ChannelSettled events

### ISSUE THREAD

**Issue by pirapira** [raiden-network/raiden#4472]
# Abstract

Now TokenNetwork.ChannelSettled() event contains locksroot of both participants.  This issue keeps track of using these locksroot values in the client.

## Motivation

If we do this, we do not have to look at the blockchain state for the locksroot values.  This eases the state pruning problem.

## Specification

Instead of asking onchain for locksroot values, use  the locksroot values found in the SettleChannel() events.

## Backwards Compatibility

This changes the state changes, so the databases will be broken.  On `raiden:develop` we don't care about that.

**Comment by LefterisJP** [comment]
Hmmm since this is a breaking DB change, if we do it, it should be done before the mainnet release.

**Comment by palango** [comment]
Theräs alreadz https://github.com/raiden-network/raiden/pull/4522 for that.

**Comment by LefterisJP** [comment]
Hmmm okay good to know this is already being worked on. But without a board entry it's hard to follow. I will add it there.

**Comment by LefterisJP** [comment]
Hmmm the assignment is wrong here. I wanted to assign Augusto. Rectifying.

**Comment by hackaugusto** [comment]
This is blocked by https://github.com/raiden-network/raiden-contracts/issues/1176
